R/model-wrappers.R

```diff
@@ -201,7 +201,7 @@ fable_model <- function(y = NULL, samples = NULL,
   return(samples)
 }
 
-#' forecastHybrid model wrapper
+#' ForecastHybrid model wrapper
```

Comment:
change back - name of package

Reply:
now I'm confused. Above you said I should correct

Reply:
I would prefer everything that isnt the name of something to be in title case as per CRAN guidelines.

Reply:
language barrier. googled title case.. :) corrected now.